### PR TITLE
#406 Update the release docs after the last release.

### LINKED ISSUE #406: Document the updated release process
I did the last release and now I am doing the 22.8.0 release.

The docs needs to be updated document how I did the release.

I am using GitHub releases and a process that is a bit similar to twisted/twisted  ... just to make it easier for me

### PR DESCRIPTION
**Pull request by adiroiban**
# Description

Fixes #406

Just the updates of the docs based on  the steps followed during the last release.

Added the release docs to the normal docs so that we can preview the rendering via RTD.

### CODE REVIEW

File: RELEASE.rst [hunk 1]
```diff
@@ -1,96 +1,102 @@
...
-- Run ``venv/bin/towncrier build --yes`` to build the the newsfragments into the release notes document and to automatically remove the newsfragment files.
+* *Choose a tag*: Type `19.9.0rc1` and select `Create new tag on publish.`
+* *Target*: Search for the release branch and select it.
+* *Title*: "Towncrier 19.9.0rc1".
+* Set the content based on the NEWS file.
+* Make sure to mark **This is a pre-release**.
+* Click `Publish release`
 
-- Commit and push to the primary repository, not a fork.
+This will trigger the PyPi release candidate.
```
Comment (hynek) [inline RELEASE.rst]:
```suggestion
This will trigger the PyPI release candidate.
```

File: RELEASE.rst [hunk 2]
```diff
@@ -1,96 +1,102 @@
...
 
-  - It is important to not use a fork so that pushed tags end up in the primary repository, server provided secrets for publishing to PyPI are available, and maybe more.
+Wait for the PyPi version to be published and then request a review for the PR from the `twisted/twisted-contributors` team.
```
Comment (hynek) [inline RELEASE.rst]:
```suggestion
Wait for the PyPI version to be published and then request a review for the PR from the ``twisted/twisted-contributors`` team.
```

File: RELEASE.rst [hunk 3]
```diff
@@ -1,96 +1,102 @@
...
 
-- If working on the first release candidate from this branch, create a PR named in the form ``Release 19.9.0``.
+In the PR request, you can give the link to the PyPi download and the documentation pages.
```
Comment (hynek) [inline RELEASE.rst]:
```suggestion
In the PR request, you can give the link to the PyPI download and the documentation pages.
```

**Review by hynek — CHANGES_REQUESTED** [review]
Some nits, can't say much about the content.

But the docs build [is failing](https://readthedocs.org/projects/towncrier/builds/17930483/) due to this line:

> /home/docs/checkouts/readthedocs.org/user_builds/towncrier/envs/407/bin/python -m pip install --upgrade --no-cache-dir pillow mock==1.0.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<2.2 jinja2<3.1.0


I'm not sure where those pins are coming from – is there something in the web interface? (Admin → Advanced Settings → Default settings)

The problem is that it uses an ancient sphinx which needs a `jinja2<3` pin (and `markupsafe<2.1`).

**Comment by adiroiban** [comment]
Thanks for the review. 

I have applied your suggestions.

The content is more for the next maintainer to know how the previous releases were done.

I tried to add a tox to build the docs on your local system... and cleaned some docs errors.

It looks like GitHub CODEOWNER PR  review only works for the first review request, and not for re-review request.

This is ready for another review.

The docs in HTML format https://towncrier--407.org.readthedocs.build/en/407/release.html

**Review by hynek — APPROVED** [review]

**Pull request merged**